uberon.obo

```diff
@@ -283690,15 +283798,18 @@ inverse_of: positively_regulated_by ! positively regulated by
 [Typedef]
 id: postaxialmost_part_of
 name: postaxialmost part of
+namespace: spatial
 xref: BSPO:0001115
 is_a: part_of ! part_of
 
 [Typedef]
 id: posterior_to
 name: posterior to
 name: posterior_to
+namespace: spatial
 def: "x posterior to y iff x is further along the antero-posterior axis than y, towards the body/tail. An antero-posterior axis is an axis that extends through an organism from head end to opposite end of body or tail." [BSPO:cjm]
 def: "x posterior_to y iff x is further along the antero-posterior axis than y, towards the body/tail. An antero-posterior axis is an axis that bisects an organism from head end to opposite end of body or tail." [http://orcid.org/0000-0002-6601-2165]
+synonym: "caudal to" NARROW vertebrate []
```

Comment:
@anitacaron @matentzn here's the term, when searching in uberon-edit file, I only see the term as a BSPO term

Reply:
Yeah, I see it in BSPO when the ontology changed to ODK in [this PR](https://github.com/obophenotype/biological-spatial-ontology/pull/30) but haven't updated the import in uberon since 2020.

Reply:
Not sure if we need to deal with it, I've approved the PR but maybe best to also get a thumbs up from @matentzn regarding this

Reply:
Its silly but not harmful. thumbs up.